app/Controller/PagesController.php

```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
+	public $uses3 = array();
```

Comment:
[staging] [code_sniffer] Spaces must be used to indent lines; tabs are not allowed